server/views/pages/create/additionalLicenceConditionInput.njk

```diff
@@ -19,49 +19,59 @@
-        <div class="govuk-grid-row">
-            <div class="govuk-grid-column-full">
-                <p class="govuk-body">
-                    {{ config.text }}
-                </p>
-                {% if config.subtext %}
-                    <div class="govuk-hint">{{ config.subtext }}</div>
-                {% endif %}
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full">
+                    <p class="govuk-body">
+                        {{ config.text }}
+                    </p>
+                    {% if config.subtext %}
+                        <div class="govuk-hint">{{ config.subtext }}</div>
+                    {% endif %}
 
-                {{ formBuilder(licence.id, config, additionalCondition, validationErrors, formResponses, csrfToken) }}
-
-                {{ govukButton({
-                    text: "Continue",
-                    type: "submit",
-                    classes: 'govuk-!-margin-top-6',
-                    attributes: { 'data-qa': 'continue' }
-                }) }}
+                    {{ formBuilder(licence.id, config, additionalCondition, validationErrors, formResponses, csrfToken) }}
+                    <div class="govuk-button-group">
+                        {{ govukButton({
+                            text: "Continue",
+                            type: "submit",
+                            classes: 'govuk-!-margin-top-6',
+                            attributes: { 'data-qa': 'continue' }
+                        }) }}
+                        {% if fileUpload %}
+                            <a class="govuk-link govuk-link--no-visited-state"
+                               href={{ "/licence/create/id/" + licence.id +"/additional-licence-conditions/condition/"+additionalCondition.code+"/file-uploads?fromReview=true" }}>Cancel</a>
+                        {% endif %}
+                    </div>
+                </div>
             </div>
-        </div>
-    </form>
+        </form>
+    </div>
 
-    <form action="{{ "/licence/create/id/" + licence.id + "/additional-licence-conditions/condition/" + additionalCondition.id + "/delete" + fromReview }}" method="POST">
-        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-        {{ govukButton({
-            text: "Delete this condition",
-            type: "submit",
-            classes: 'govuk-button--secondary',
-            attributes: { 'data-qa': 'delete' }
-        }) }}
-    </form>
+    {% if fileUpload == false %}
+        <form action="{{ "/licence/create/id/" + licence.id + "/additional-licence-conditions/condition/" + additionalCondition.id + "/delete" + fromReview }}"
+              method="POST">
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+            {{ govukButton({
+                text: "Delete this condition",
+                type: "submit",
+                classes: 'govuk-button--secondary',
+                attributes: { 'data-qa': 'delete' }
```

Comment:
Might be worth generally adding `preventDoubleClick` attributes to buttons that do stuff?